bin/compile

```diff
@@ -18,20 +18,18 @@ BUILDPACK_DIR="$(dirname $(dirname $0))"
 
 # config
 S3_BUCKET="ci-database-binary"
-POSTGRESQL_MAJOR_VERSION=16
+POSTGRESQL_MAJOR_VERSION=17
 if [ -f "$ENV_DIR/POSTGRESQL_VERSION" ]
 then
   POSTGRESQL_MAJOR_VERSION="$(cat $ENV_DIR/POSTGRESQL_VERSION)"
 fi
 
 case "$POSTGRESQL_MAJOR_VERSION" in
+  17)  POSTGRESQL_VERSION="17.5";;
```

Comment:
Should we update the other minor versions while we are making this change?

Reply:
+1. here's what I see (https://www.postgresql.org/docs/release/):
16.9
15.13
14.18
13.21